eng/restore/harvestPackages/harvestPackages.depproj

```diff
@@ -1,47 +1,32 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project InitialTargets="AddPackageReferences" Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- Baselining errors regarding net461 assets getting picked when targeting netstandard20 -->
+    <NoWarn>$(NoWarn);NU1701</NoWarn>
   </PropertyGroup>
 
-  <Import Project="harvestPackages.props" />
-
   <Import Project="$(RepoRoot)pkg/baseline/baseline.props" />
 
-  <!--
-    Custom target to update the stable versions. Ideally we would just run this target as part of the build but
-    currently the restore step happens in CLI which doesn't have matching dependencies for the package task. Currently
-    they have different versions of nuget libraries. We can look at fixing that in the future but for now this will need
-    to be manually run to update the harvestPackages.props file.
-  -->
-  <Target Name="UpdateToLatestStablePackages">
+  <!-- only restore this project during the build, don't copy any of it's packages
+       The sole purpose of this project is to download packages that can be examined
+       for harvesting binaries & support. -->
+  <Target Name="Build" DependsOnTargets="RestorePackages" />
+
+  <Target Name="AddPackageReferences">
     <ItemGroup>
-      <PackageReference>
-        <!-- Set version to highest version possible to ensure we get the highest stable package version -->
-        <Version>255.255.255</Version>
-      </PackageReference>
+      <_AllPkgProjs Include="$(RepoRoot)src/**/*.pkgproj" />
```

Comment:
Out of curiosity, do you have numbers how long this takes? Usually globbing is very slow in msbuild.

Reply:
mm not sure but I don't think this is significant as I definitely can't notice this locally and we are doing globing all over the place in our build.

Reply:
This will be dwarfed by the cost of the restore.